### ISSUE TITLE
Transparent dark background crashes on 6.0

### ISSUE THREAD

**Issue by vlopezj** [ligi/FAST#69]
To reproduce:
Select "Transparent dark" as Theme in the application settings. Alternatively, set the theme to "Transparent dark" in 5.9, and upgrade to 6.0.

Expected behaviour:
The selection is displayed in the settings, and the background becomes transparent.

Observed behaviour:
The application crashes. If the application is restarted, it will crash again.
The following trace turns up in the log:

```
2015-01-07 06:22:13,ERROR,ActivityThread,Unknown,Failed to inflate
android.view.InflateException: Binary XML file line #16: Error inflating class android.widget.EditText
	at android.view.LayoutInflater.createView(LayoutInflater.java:620)
	at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:669)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:694)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
	at de.robv.android.xposed.XposedBridge.invokeOriginalMethodNative(Native Method)
	at de.robv.android.xposed.XposedBridge.handleHookedMethod(XposedBridge.java:631)
	at android.view.LayoutInflater.inflate(Native Method)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:353)
	at com.android.internal.policy.impl.PhoneWindow.onIntChanged(PhoneWindow.java:1379)
	at com.android.internal.policy.impl.PhoneWindow.updateInt(PhoneWindow.java:3606)
	at com.android.internal.policy.impl.PhoneWindow.setFeatureInt(PhoneWindow.java:1320)
	at org.ligi.fast.ui.SearchActivity.onCreate(Unknown Source)
	at android.app.Activity.performCreate(Activity.java:5231)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2169)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2271)
	at android.app.ActivityThread.access$800(ActivityThread.java:144)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1205)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:136)
	at android.app.ActivityThread.main(ActivityThread.java:5146)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:732)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:566)
	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.constructNative(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at android.view.LayoutInflater.createView(LayoutInflater.java:594)
	... 30 more
Caused by: android.content.res.Resources$NotFoundException: Resource is not a ColorStateList (color or path): TypedValue{t=0x2/d=0x1010098 a=-1}
	at android.content.res.Resources.loadColorStateList(Resources.java:2376)
	at android.content.res.TypedArray.getColorStateList(TypedArray.java:343)
	at android.content.res.XResources$XTypedArray.getColorStateList(XResources.java:912)
	at android.widget.TextView.<init>(TextView.java:960)
	at android.widget.EditText.<init>(EditText.java:60)
	at android.widget.EditText.<init>(EditText.java:56)
	... 33 more
```

**Comment by wvengen** [comment]
I'm seeing this as well.

**Comment by krt16s** [comment]
An issue was raised on FDroid's tracker, but since we haven't changed anything and you two are experienceing this issue as well, it's most likely nothing on our side. @wvengen, @vlopezj: You are using the play-store version?

For reference: https://gitlab.com/fdroid/fdroiddata/issues/140

**Comment by vlopezj** [comment]
I'm using the F-Droid version (6.0), F-Droid 5.9 works all right.
I downloaded the Play Store version on my computer, and tested it on my phone. It works, but it's showing as "5.9" on the play store; at least when accessing from the computer.
https://play.google.com/store/apps/details?id=org.ligi.fast
![fast](https://cloud.githubusercontent.com/assets/1593773/5695002/29ec529e-9988-11e4-89aa-c87cabd341b2.png)

**Comment by wvengen** [comment]
I'm using the F-Droid version, 6.0.

**Comment by ligi** [comment]
thanks for reporting and commenting!
the issue is fixed in 6.1 - weird problem though - the style looked legit. Uploaded the version to play as beta - fdroid will come a bit later - this version also includes the nice UI-improvements by @cketti

**Comment by wvengen** [comment]
Works on 6.1 (F-Droid). Thanks for fixing!
